File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioDtoLayoutLocmapPin.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnMapPinShape;
import com.neome.java.api.meta.base.Types.MetaIdField;
import com.neome.java.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioDtoLayoutLocmapPin extends StudioBase
{
  @Nullable
  public FieldDtoImage avatar;

  @Nullable
  public MetaIdField avatarFieldId;

  @Nullable
  public MetaIdVar avatarVarId;

  @Nullable
  public StudioDtoColor color;

  @Nullable
  public MetaIdField colorFieldId;

  @Nullable
  public MetaIdVar colorVarId;

  @Nullable
  public String label;

  @Nullable
  public MetaIdField labelFieldId;

  @Nullable
  public StudioValueVarIdText labelVarId;

  @Nullable
  public EnumDefnMapPinShape shape;

  @Nullable
  public MetaIdField shapeFieldId;

  @Nullable
  public MetaIdVar shapeVarId;

  @Nullable
  public String toolTip;

  @Nullable
  public MetaIdField toolTipFieldId;

  @Nullable
  public StudioValueVarIdParagraph toolTipVarId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoLayoutLocmapPin.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnMapPinShape
import com.neome.api.meta.base.dto.FieldDtoImage
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.meta.base.dto.StudioBase
import com.neome.api.meta.base.dto.StudioDtoColor
import com.neome.api.meta.base.dto.StudioValueVarIdParagraph
import com.neome.api.meta.base.dto.StudioValueVarIdText

interface StudioDtoLayoutLocmapPin : StudioBase
{
  val avatar: FieldDtoImage?
  val avatarFieldId: MetaIdField?
  val avatarVarId: MetaIdVar?
  val color: StudioDtoColor?
  val colorFieldId: MetaIdField?
  val colorVarId: MetaIdVar?
  val label: String?
  val labelFieldId: MetaIdField?
  val labelVarId: StudioValueVarIdText?
  val shape: EnumDefnMapPinShape?
  val shapeFieldId: MetaIdField?
  val shapeVarId: MetaIdVar?
  val toolTip: String?
  val toolTipFieldId: MetaIdField?
  val toolTipVarId: StudioValueVarIdParagraph?
}